### PR TITLE
minipro: update to version 0.7

### PR DESCRIPTION
**Pull request by MaximeChretien**
Also includes:
- manpage support fix
- data location fix
- bash completion support

### CODE REVIEW

File: sci-electronics/minipro/minipro-0.7.recipe [hunk 1]
```diff
@@ -48,13 +48,13 @@ BUILD_PREREQUIRES="
 
 BUILD()
 {
-	DEST_DIR=$installDestDir PREFIX=$prefix SHARE_DIR=$relativeDataDir make $jobArgs
-	DEST_DIR=$installDestDir PREFIX=$prefix SHARE_DIR=$relativeDataDir make library $jobArgs
+	DEST_DIR=$installDestDir PREFIX=$prefix SHARE_DIR=$relativeDataDir/minipro make $jobArgs
+	DEST_DIR=$installDestDir PREFIX=$prefix SHARE_DIR=$relativeDataDir/minipro make library $jobArgs
```
Comment (jmairboeck) [inline sci-electronics/minipro/minipro-0.7.recipe:52]: DEST_DIR is not needed, except when cross-compiling, I think.

File: sci-electronics/minipro/minipro-0.7.recipe [hunk 2]
```diff
@@ -48,13 +48,13 @@ BUILD_PREREQUIRES="
...
 }
 
 INSTALL()
 {
-	DEST_DIR=$installDestDir PREFIX=$prefix BIN_DIR=$relativeBinDir SHARE_DIR=$relativeDataDir MAN_DIR=$relativeManDir make install
+	DEST_DIR=$installDestDir PREFIX=$prefix BIN_DIR=$relativeBinDir SHARE_DIR=$relativeDataDir/minipro MAN_DIR=$relativeManDir/man1 COMPLETIONS_DIR=$prefix/$relativeDataDir/bash-completion/completions make install
```
Comment (jmairboeck) [inline sci-electronics/minipro/minipro-0.7.recipe:57]: * Same here about DEST_DIR.
 * These lines could really use some line-breaks, they are way over 100 characters.
 * `$prefix/$relativeDataDir` can be replaced with `$dataDir`.

**Review by jmairboeck — COMMENTED** [review]
Just some cleanup and formatting comments

File: sci-electronics/minipro/minipro-0.7.recipe (hunk 1 above)
Comment (MaximeChretien) [reply sci-electronics/minipro/minipro-0.7.recipe]: Alright thanks, I was not sure about it.

File: sci-electronics/minipro/minipro-0.7.recipe (hunk 2 above)
Comment (MaximeChretien) [reply sci-electronics/minipro/minipro-0.7.recipe]: I do agree that it looks better with some formatting. I also overlooked the dataDir when looking at the wiki.

**Comment by Begasus** [comment]
@jmairboeck I'll let you do the follow ups on this one. :)

**Review by jmairboeck — APPROVED** [review]

**Pull request merged**